CONTRIBUTING.md

```diff
@@ -45,4 +45,67 @@ on how to build iceberg locally.
 
 For Java styling, check out the section
 [Setting up IDE and Code Style](https://iceberg.apache.org/community/#setting-up-ide-and-code-style) from the
-documentation site. For Python, please use the tox command `tox -e format` to apply autoformatting to the project.
+documentation site.
+
+For Python, please use the tox command `tox -e format` to apply autoformatting to the project.
+
+### Java style guidelines
+
+#### Line breaks
+
+Continuation indents are 2 indents (= 4 spaces) from the start of the previous line.
+
+Try to break long lines at the same semantic level to make code more readable.
+* Don't use the same level of indentation for arguments to different methods
```

Comment:
I do not understand this sentence. Maybe, just my limited English, but some rewording might help.
Thanks, Peter

Reply:
Or an example

Reply:
The code block right below is the example. `doSomething` and the `ArgumentClass` constructor are two different methods and in the `BAD:` example you can see the arg `2` for `ArgumentClass` and the arg `3` for `doSomething` are both indented at the same level, making it hard at first glance to notice which argument belongs to which method.

Reply:
Shall we add some chained methods to the example as well? Or when this will be displayed on the site, it will be easier to see the connection between the example and the code? Github view doesn't really help here.

Reply:
Sure, will do.

Reply:
thx!

Reply:
Added below.